### DISCUSSION TITLE
Atlas connection issue

### DISCUSSION THREAD

**Discussion by Jovannino** [Budibase/budibase#12175] [General]
Hello,
I was trying as first time to use budibase.
I have to connect to Atlas MongoDb database that has this url as replica set

mongodb://[10.198.71.184:27017](http://10.198.71.184:27017/),[10.198.71.184:27018](http://10.198.71.184:27018/),[10.198.71.184:27019/T4H-sample?replicaSet=rs0](http://10.198.71.184:27019/T4H-sample?replicaSet=rs0)

but it doesn't work .
How can I do a test ?

Thanks
Giovanni

**Comment by melohagan** [comment]
Hi @Jovannino 

The format of the URL doesn't appear as how I normally connect, which is via `mongodb+srv:// ... mongodb.net`
